PresentData/PresentMonTraceConsumer.cpp

```diff
@@ -1191,94 +1188,84 @@ void PMTraceConsumer::RemovePresentFromTemporaryTrackingCollections(std::shared_
     // mPresentsByProcessAndSwapChain, and mAllPresents.
 
     // mPresentByThreadId
+    auto threadEventIter = mPresentByThreadId.find(p->ThreadId);
+    if (threadEventIter != mPresentByThreadId.end() && threadEventIter->second == p) {
+        mPresentByThreadId.erase(threadEventIter);
+    }
+    if (p->DriverBatchThreadId != 0)
     {

```

Comment:
Nitpick: Formatting.

Reply:
Kept these together since they were both about mPresentByThreadId

Reply:
I'm a bit of a toss up on this, but added the spacing